src/main/network/http.ts

```diff
@@ -281,35 +294,30 @@ export const httpGetWithAuth =
             const options = _options || {};
             options.method = "get";
 
-            const response = await httpFetchFormattedResponse(
+            if (enableAuth) {
+                const url = _url instanceof URL ? _url : new URL(_url);
+                const { host } = url;
+
+                const auth = await getConfigRepoOpdsAuthenticationToken(host);
+
+                if (
+                    typeof auth === "object"
+                    && auth.accessToken
+                ) {
+                    // We have an authentication token for this host.
+                    // We should use it by default
+                    // Because we won't always get a 401 response that will ask us to use it.
+                    return httpGetUnauthorized(auth)(_url, options, _callback, ..._arg);
+                }
```

Comment:
why the else is missing ?

Reply:
handleCallback isn't called and returns both body and response object to ipc serializer